app/services/tag_archiver.rb

```diff
@@ -56,4 +57,23 @@ def republish_tag
     presenter = TagPresenter.presenter_for(tag)
     ContentItemPublisher.new(presenter).send_to_publishing_api
   end
+
+  def unsubscribe_from_email_alerts
+    EmailAlertsUnsubscriber.call(
+      item: tag,
+      body: unsubscribe_email_body,
+    )
+  end
+
+  def unsubscribe_email_body
+    <<~BODY
```

Comment:
I've been out of the loop - does the body need a content team pass?

Reply:
The email copy is in the [doc](https://docs.google.com/document/d/1ixKUkQQV8OD856B4Bfy_ZxpQFrqM0GaLDumesPspWbo/edit) but I asked Kati a question.